src/sql/workbench/browser/designer/designer.ts

```diff
@@ -91,19 +95,22 @@ export class Designer extends Disposable implements IThemable {
 	private _tableCellEditorFactory: TableCellEditorFactory;
 	private _propertiesPane: DesignerPropertiesPane;
 	private _buttons: Button[] = [];
+	private _actions: IAction[] = [];
 	private _inputDisposable: DisposableStore;
 	private _loadingTimeoutHandle: any;
 	private _groupHeaders: HTMLElement[] = [];
 	private _issuesView: DesignerIssuesTabPanelView;
 	private _scriptEditorView: DesignerScriptEditorTabPanelView;
+	private _taskbar: Taskbar;
```

Comment:
there are multiple task bars in the designer (one per table), why do we only have one taskbar at the designer level?